include/AudioBufferView.h

```diff
@@ -153,6 +153,14 @@ class InterleavedBufferView : public detail::BufferViewData<SampleT, channelCoun
 		}
 	}
 
+	//! @return a subview at the given frame offset `offset` with a frame count of `frames`
+	constexpr auto subspan(f_cnt_t offset, f_cnt_t frames) const
+	{
+		assert(offset <= this->m_frames);
+		assert(offset + frames <= this->m_frames);
+		return InterleavedBufferView<SampleT>{this->m_data + offset * this->channels(), this->channels(), frames};
```

Comment:
```suggestion
	constexpr auto subspan(f_cnt_t offset, f_cnt_t frames) const -> InterleavedBufferView<SampleT, channelCount>
	{
		assert(offset <= this->m_frames);
		assert(offset + frames <= this->m_frames);
		return {this->m_data + offset * this->channels(), this->channels(), frames};
```
This returns a buffer view with the same `channelCount` instead of converting it to one with `DynamicChannelCount`.